cudax/include/cuda/experimental/__async/type_traits.cuh

```diff
@@ -32,19 +32,19 @@ namespace cuda::experimental::__async
 {
 
 template <class _Ty>
-using __remove_ref_t = _CUDA_VSTD::__libcpp_remove_reference_t<_Ty>;
+using __remove_ref_t = _CUDA_VSTD::remove_reference_t<_Ty>;
 
 //////////////////////////////////////////////////////////////////////////////////////////////////
-// __decay_t: An efficient implementation for ::std::decay
+// decay_t: An efficient implementation for ::std::decay
 #if defined(_CCCL_BUILTIN_DECAY)
 
 template <class _Ty>
-using __decay_t = __decay(_Ty);
+using decay_t = __decay(_Ty);
```

Comment:
best to leave this one alone. `cuda::std::decay_t` still forces the instantiation of the `cuda::std::decay` class template, and i'm trying to avoid that.

Reply:
While it would be nice, this would only be possible for clang-cuda because nvcc cannot handle `__decay` due to some libstdc++ interference

Reply:
Reverted to old definition